prow/plugins/verify-owners/verify-owners.go

```diff
@@ -46,8 +47,17 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	configString := `The verify-owners plugin does not have any blacklisted label to watch for.`
```

Comment:
This might be confusing for someone. Can we pass no config if nothing is configured?